docs/_posts/gadde5300/2021-12-10-bert_hi_en_ner_hi.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: model
+title: NER Model for Hindi+English
+author: John Snow Labs
+name: bert_hi_en_ner
+date: 2021-12-10
+tags: [ner, hi, en, open_source]
+task: Named Entity Recognition
+language: hi
+edition: Spark NLP 3.2.0
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model was imported from Hugging Face to carry out Name Entity Recognition with mixed Hindi-English texts. This model is trained using the LinCE repository.
```

Comment:
This model was imported from Hugging Face to carry out Name Entity Recognition with mixed Hindi-English texts, provided by the LinCE repository.